scripts/refresh_spaces_counters.ts

```diff
@@ -0,0 +1,35 @@
+import 'dotenv/config';
+import db from '../src/helpers/mysql';
+
+// Usage: yarn ts-node scripts/refresh_spaces_counters.ts
+async function main() {
+  const spaces = await db.queryAsync(`SELECT id FROM spaces`);
+
+  console.log(`Found ${spaces.length} spaces`);
+
+  for (const i in spaces) {
+    const stats = await db.queryAsync(
+      `SELECT COUNT(voter) as vote_count, COUNT(DISTINCT(proposal)) as proposal_count FROM votes WHERE space = ? GROUP BY space`,
+      [spaces[i].id]
```

Comment:
I think this ignores proposals with `0` votes 

Maybe we can add a separate query to the `proposals` table